cfn-resources/serverless-instance/README.md

```diff
@@ -0,0 +1,61 @@
+# MongoDB::Atlas::ServerlessInstance
+
+## Description
+Returns, adds, edits, and removes serverless instances.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Local Testing
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/project
+./test/project.create-sample-cfn-request.sh YourProjectName > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
+```
+
+Both CREATE & DELETE tests must pass.
+
+## Installation
+TAGS=logging make
+cfn submit --verbose --set-default
+
```

Comment:
Missing section to examples/serverless-instance folder so customers know to use it